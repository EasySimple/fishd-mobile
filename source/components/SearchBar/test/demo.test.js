import webDemoTest from '../../../tests/demoTest';

webDemoTest('SearchBar');
