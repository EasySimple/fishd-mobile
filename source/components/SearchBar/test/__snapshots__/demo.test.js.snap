// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders SearchBar demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-searchbar"
      >
        <div
          class="fm-searchbar__content"
        >
          <div
            class="fm-searchbar__icon"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-search"
              style="width:14px;height:14px;color:#333"
            />
          </div>
          <div
            class="fm-searchbar__body"
          >
            <input
              class="fm-searchbar__input"
              placeholder="请输入搜索关键词"
              type="search"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
